dashboard/app.py

```
@@ -56,7 +56,74 @@
+    min-height: 300px !important;
+    height: auto !important;
+}
+uk-chart .apexcharts-canvas {
+    width: 100% !important;
+    height: 100% !important;
+}
+        """),
+        Script("""
+function initializeCharts() {
+    const elements = document.querySelectorAll('uk-chart:not([data-chart-initialized])');
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)